build-tools/automation/azure-pipelines.yaml

```diff
@@ -347,9 +347,9 @@ extends:
           submodules: recursive
 
         - task: UseDotNet@2
-          displayName: Install .NET 9.x
+          displayName: Install .NET 10.x
           inputs:
-            version: 9.x
+            version: 10.x
```

Comment:
Similar to how we declare `DotNetTargetFramework` and `DotNetStableTargetFramework` in the variables template, maybe we should do something similar for `DotNetSdkVersion` to avoid needing to bump this again separately at some point in the future?

https://github.com/dotnet/android/blob/main/build-tools/automation/yaml-templates/variables.yaml#L26

Good with this as is to unblock things though